uftrace.h

```diff
@@ -30,6 +30,8 @@
 #define KB 1024
 #define MB (KB * 1024)
 
+#define BUF_SIZE 4096
```

Comment:
In many cases (usually dealing with a file path) it's more natural to use `PATH_MAX` instead IMO.